source/tutorials/openmm.rst

```diff
@@ -17,4 +23,129 @@ folder of the GitHub repository. It contains:
   NanoVer simulation using a python client.
 * `openmm_neuraminidase`: A notebook that demonstrates how to set up an interactive
   OpenMM simulation from scratch with NanoVer and change the visualisations of
-  different atom selections for a protein-ligand system.
\ No newline at end of file
+  different atom selections for a protein-ligand system.
+
+Openmm XML Files
+==============================================
+
+The XML format described here is used specifically for manipulating NanoVer simulations from OpenMM serialized XML files. It consists of a root element ``<OpenMMSimulation>`` containing three main components:
```

Comment:
```suggestion
The XML format described here is used specifically for saving and loading NanoVer OpenMM simulations. It consists of a root element ``<OpenMMSimulation>`` containing three main components:
```
The suggested change here reflects the fact that the xml file also contains non-OpenMM components from the PDB file.